lib/turn.rb

```diff
@@ -70,4 +44,29 @@ def record_move_print_board(move, player)
     @board.invert_board_diagonal
     @board.print_board
   end
+
+   #Helper Methods valid_input_character? and is_move_invalid? evaluate player input
+
+   def valid_input_character?(move)
+    ((move.is_a?(Integer) == false) && ((@board.game_board).has_key?(move.upcase)))
+  end
+    
+  def is_move_invalid?(move)
```

Comment:
I believe the convention for methods ending in a `?` to return a boolean value. This method doesn't return a boolean value, so I think it would be appropriate to remove the `?` at the end.